docs/release-notes/1.6.4.md

```diff
@@ -0,0 +1,3 @@
+## New Features
```

Comment:
Previously patch release notes weren't added on main branch, only on release

Reply:
tbh I would like to have everything on main and distributed to patch release branches, but lets keep it consistent like until now.